docs/commands/env.md

```diff
@@ -175,15 +175,18 @@ netlify env:set
 - `http-proxy` (*string*) - Proxy server address to route requests through.
 - `http-proxy-certificate-filename` (*string*) - Certificate file to use when connecting using a proxy server
 - `scope` (*builds | functions | post-processing | runtime*) - Specify a scope (default: all scopes)
+- `secret` (*boolean*) - Indicate whether the environment variable value can be read again.
```

Comment:
Always worried to suggest you can't read the value because depending on if the dev is thinking about reading from a code standpoint vs a UI/API standpoint can cause confusion.

```suggestion
- `secret` (*boolean*) - Indicate whether this value should be treated as write-only when outside of code running on Netlify.
```